api/grpcserver/grpcserver_test.go

```diff
@@ -452,7 +455,7 @@ func launchServer(t *testing.T, services ...ServiceAPI) func() {
 
 	return func() {
 		require.NoError(t, jsonService.Close())
-		grpcService.Close()
+		_ = grpcService.Close()
```

Comment:
Maybe add a comment here saying something like,

`// We explicitly ignore errors that occur during shutdown`